app/interactions/add_dragon.rb

```diff
@@ -4,6 +4,8 @@ class AddDragon < ActiveInteraction::Base
 
   def execute
     dragon.user = user
+    stat = Stat.new(dragon: dragon, strength: 0, luck: 0, perception: 0)
+    stat.save
```

Comment:
Wydaje mi się, że to nie będzie działać, bo zapisujemy `Stat` w momencie kiedy dragon nie ma jeszcze ID

To jest ten case, kiedy warto użyć `stat.save!` (z wykrzyknikiem), bo inaczej nie zobaczymy błędu

Reply:
```
def execute
    dragon.user = user
    pay_for_dragon if dragon.save!
    Stat.create!(dragon: dragon, strength: 0, luck: 0, perception: 0)
end
```
to powinno działać OK